docs/developer/endpoints/public-rpc.mdx

```diff
@@ -3,17 +3,31 @@ slug: /public-rpc
 title: Lava Public RPC 🌋
 ---
 
-# Lava Public RPC (Testnet-2)
+import CommunityMainnetEndpointsTable from './components/community-rpc/public-rpc-table.jsx'
 
-## Official Endpoints ⭕
+# Lava Public RPC
 
-| Service 🔌 | URL 🔗 | Protocol |
-|--------|-------|-------- |
-| 🟢 rpc | <https://lav1.tendermintrpc.lava.build:443>    | HTTP/1.1 |
-| 🟢 websocket | lav1.tendermintrpc.lava.build/websocket    | HTTP/1.1 |
-| 🟢 rest | <https://lav1.lava.build:443>    | HTTP/1.1 |
-| 🟢 grpc | lav1.grpc.lava.build:443 |    HTTP/2 |
-| 🟢 archive/rpc | <https://lava-archive.lavanet.xyz:443> |    HTTP/1.1 |
+## Official Testnet-2 Endpoints ⭕
+
+| Service 🔌      | URL 🔗                                         | Protocol | Notes |
+|-----------------|------------------------------------------------|----------|-------- |
+| 🌐 RPC          | <https://lava-archive.lavanet.xyz:443>         | HTTP/1.1 | Archive |
+| 🌐 RPC          | <https://lav1.tendermintrpc.lava.build:443>    | HTTP/1.1 | Archive |
+| 📡 WebSocket    | wss://lav1.tendermintrpc.lava.build/websocket  | HTTP/1.1 | Archive |
+| 🔗 REST API     | <https://lav1.lava.build:443>                  | HTTP/1.1 | Archive |
+| ⚡ gRPC         | lav1.grpc.lava.build:443                       | HTTP/2   | Archive |
+
+## Official Mainnet Endpoints 🌎
+
+Official endpoint with Lava over Lava:
+
+```bash
+curl -X GET -H "Content-Type: application/json" https://lava.rest.lava.build/cosmos/base/tendermint/v1beta1/blocks/latest
+```
```

Comment:
```suggestion
```

No need. Maybe in a different section for "exmaple" usage.

Reply:
Lines deleted